Amazon.IonObjectMapper/TypeAnnotation/DefaultAnnotationConvention.cs

```diff
@@ -0,0 +1,25 @@
+﻿/*
```

Comment:
How come these files don't have C# syntax highlighting?

Reply:
I can't seem to reproduce the issue.